src/components/Media/Pagination.tsx

```
@@ -52,10 +56,28 @@ export function Pagination({
+      setJumpError(t('mediaPage.pageRangeError', 'Page must be 1-{{max}}', { max: totalPages }))
+      return
     }
+    // Clamp to current totalPages and clear error + input on success
+    const clampedPage = Math.min(pageNum, totalPages)
+    setJumpError('')
+    setJumpToPage('')
+    onPageChange(clampedPage)
+  }
+
+  // Clear error when totalPages changes (pagination state updated)
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)